clients/pkg/promtail/targets/cloudflare/target.go

```diff
@@ -28,6 +28,8 @@ import (
 // The minimun window size is 1 minute.
 const minDelay = time.Minute
 
+const cloudflareTooEarlyError = "invalid time range: too early: logs older than 168h0m0s are not available"
```

Comment:
ah maybe I misread the initial messages in this PR, I think we could just remove `too early` and say `invalid time range: logs older than 168h0m0s are not available` or `invalid start time: logs older than 168h0m0s are not available`

Reply:
Too early is the message from cloudlfare we're matching on so no cannot be anything else. I did remove invalid start time so that it is tighter.